www/source/docs/best-practices.html.md.erb

```diff
@@ -4,7 +4,6 @@ This chapter covers recommended best practices for runtime and buildtime.
 We focus on best practices for packages that use Habitat Builder for continuous builds.
 
 ## Table of Contents
-  - [Quickstart Guide](#quickstart-guide)
```

Comment:
the link also needs to be removed from the sidebar

Reply:
@tashimi I don't know find where sidebar contents are written :/ I can find the sidebar layout, but not the content...

Reply:
I've found it. Done ;)